check_manifest.py

```diff
@@ -340,6 +355,9 @@ def _list_submodules(cls):
         # there's no .gitmodules file.
         if not os.path.exists('.gitmodules'):
             return []
+        # Bug: git submodule will print /paths/like/this when you run it from
+        # an msys shell (like Git Bash), and we will fail then.  See
+        # https://github.com/mgedmin/check-manifest/issues/64
```

Comment:
So this is a bit misleading: the wrong paths will happen if you run an msys shell and then clear `MSYSTEM` from the environment (like `tox` does).